### ISSUE TITLE
psmeca -Sz flag GMT6

### ISSUE THREAD

**Issue by sabertwirl** [GenericMappingTools/gmt#1070]
It seems the following special moment tensor and maybe others too make problems with -Sz flag. At first -Sz considers the two coordinate numbers of beachball placement as text string and secondly the nodal lines goes out of the beachball to where it seems to be the coordinate (0,0). 

I had problems with -L flag in GMT5, that why I tried GMT6.

```
echo 41.0  21.0  10  0.435135  0.263431  0.094889  0.252481  -0.318408  -1.000000  0  0  0 | psmeca -R39/45/20/35 -JX30/30 -Sz3.15 -Gred@50 -M -L0.6p,red > 01.ps
```

The resulted output:
![01](https://user-images.githubusercontent.com/52400174/60419852-1ec56580-9b9b-11e9-82ec-0bfac2635c58.png)

* Operating system: Windows 10
* Version of GMT: 6.0.0rc1

**Comment by PaulWessel** [comment]
There is a single point on the red line that returns to (0,0) and back again that causes this.

**Comment by seisman** [comment]
@PaulWessel I'll see if I can fix this issue tonight.

**Comment by seisman** [comment]
Fixed by #1073.
